README.md

```diff
@@ -48,116 +49,141 @@ SERVER_PRIVATE_KEY=
+## Amman Usage
+
+to develop locally with amman run:
+
+```
+amman start .ammanrc.js
+```
+
+This starts up a fresh local validator with a local storage proxy.
+to get an explorer for this local validator you can visit:
+https://amman-explorer.metaplex.com/#/clusterStats?cluster=custom&customUrl=http%3A%2F%2Flocalhost%3A8899
+
+From another terminal window run:
+
+```
+ts-node localnet/run.ts
+```
+
+add code to `setup.ts` to test whatever you are working on
+
+## Create a Production Build
 
 To create a production ready build use `yarn build`. Build files can be found in `.next`.
 
 Detailed documentation can be found at https://nextjs.org/docs/deployment#nextjs-build-api
 
 ## What can Wonka do for you?
 
-
 # Create Metaplex CandyMachine v2s
-  Send the following mutation to create a CandyMachineV2
-
-  ```
-  candyMachineUpload(
-    callbackUrl: <URL to POST to when candymachine is complete>,
-    collectionMint: <COLLECTION MINT for CANDY MACHINE>,
-    config: <JSON config compatible with SUGAR CLI, see below>,
-    env: <MAINNET or DEVNET>,
-    filesZipUrl: <URI of zip file that contains assets>,
-    keyPair: <FULL KEYPAIR>,
-    rpc: <PASS IN AN RPC URL>,
-    setCollectionMint: <BOOL, set to true if you are passing a collection>
-  )
-
-  Example JSON config
-   {
-    "price": 0.1,
-    "number": $ITEMS,
-    "symbol": "TEST",
-    "sellerFeeBasisPoints": 500,
-    "gatekeeper": null,
-    "solTreasuryAccount": "$(solana address)",
-    "splTokenAccount": null,
-    "splToken": null,
-    "goLiveDate": "$(date "+%Y-%m-%dT%T%z" | sed "s@^.\{22\}@&:@")",
-    "endSettings": null,
-    "whitelistMintSettings": null,
-    "hiddenSettings": $HIDDEN_SETTINGS,
-    "uploadMethod": "${STORAGE}",
-    "ipfsInfuraProjectId": "${INFURA_ID}",
-    "ipfsInfuraSecret": "${INFURA_SECRET}",
-    "awsConfig": {
-        "bucket": "${AWS_BUCKET}",
-        "profile": "${AWS_PROFILE}",
-        "directory": "${AWS_DIRECTORY}"
-    },
-    "nftStorageAuthToken": "${NFT_STORAGE_TOKEN}",
-    "shdwStorageAccount": $SHDW,
-    "retainAuthority": true,
-    "isMutable": true,
-    "creators": [
-    {
-      "address": "$(solana address)",
-      "share": 100
-    }
-  ]
+
+Send the following mutation to create a CandyMachineV2
+
+```
+candyMachineUpload(
+  callbackUrl: <URL to POST to when candymachine is complete>,
+  collectionMint: <COLLECTION MINT for CANDY MACHINE>,
+  config: <JSON config compatible with SUGAR CLI, see below>,
+  env: <MAINNET or DEVNET>,
+  filesZipUrl: <URI of zip file that contains assets>,
+  keyPair: <FULL KEYPAIR>,
+  rpc: <PASS IN AN RPC URL>,
+  setCollectionMint: <BOOL, set to true if you are passing a collection>
+)
+
+Example JSON config
+ {
+  "price": 0.1,
+  "number": $ITEMS,
+  "symbol": "TEST",
+  "sellerFeeBasisPoints": 500,
+  "gatekeeper": null,
+  "solTreasuryAccount": "$(solana address)",
+  "splTokenAccount": null,
+  "splToken": null,
+  "goLiveDate": "$(date "+%Y-%m-%dT%T%z" | sed "s@^.\{22\}@&:@")",
```

Comment:
this is really confusing to me. Is this the actual thing you're supposed to put in the JSON, or are you supposed to put a date in the given format?

Reply:
my autoformatter fucked with this file, but yeah, goLiveDate should be ms since epoch i believe, will fix.